learning/tour-of-beam/frontend/lib/config.dart

```diff
@@ -26,7 +26,8 @@ const cloudFunctionsBaseUrl = 'https://'
 
 // Copied from Playground's config.g.dart
 
-const String kAnalyticsUA = 'UA-73650088-2';
+const String kAnalyticsUA = 'G-349612187';
```

Comment:
Does the `usage` package work with the Analytics-4 new counters?

Reply:
The open [issue](https://github.com/dart-lang/usage/issues/185) indicates that it doesn't. I haven't found a wrapper package that supports GA4 other than firebase_analytics.

Reply:
Dropped `usage`, manually submitting to GA4.